test/unit/lodash.spec.js

```diff
@@ -0,0 +1,628 @@
+        });
+        it('should return new function', function(){
+            var runResults = _.runInContext.apply(_, params);
+            expect(runResults()).to.have.property('__wrapped__', undefined);
+            expect(runResults()).to.have.property('__actions__');
+            expect(runResults()).to.have.property('__chain__', false);
+            expect(runResults()).to.have.property('__index__', 0);
+            expect(runResults()).to.have.property('__values__', undefined);
+        });
+    });
+    
+    describe('times', function() {
+        var params;
+        before(function(){
+            params = datasets.times();
+        });
+        it('should check types', function(){
+            var elem1 = params[0],
+                elem2 = params[1];
+            expect(typeof elem1).to.eql('number');
+            expect(elem2 instanceof Function).to.eql(true);
+            expect(params).to.have.length(2);
+            expect(elem1 instanceof Array).to.eql(false);
+        });
+        it('should return array with square values', function(){
+            expect(_.times.apply(_, params)).to.eql([0, 1, 4, 9, 16]);
+            expect(_.times.apply(_, params)).to.have.length(5);
+        });
+    });
+
+    describe('toPath', function() {
+        var params;
+        before(function(){
+            params = datasets.toPath();
+        });
+        it('should check type', function(){
+            var elem1 = params[0];
+            expect(typeof elem1).to.eql('string');
+            expect(elem1).to.have.length(56);
+        });
+        it('should return array with ', function(){
+            var toPathResults = _.toPath.apply(_, params);
+            expect(toPathResults).to.eql(['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday']);
+            expect(toPathResults).to.have.length(7);
+        });
+    });
+    
+    describe('uniqueId', function() {
+        describe('uniqueId1', function() {
+            var params;
+            before(function(){
+                params = datasets.uniqueId1();
+            });
+            it('should check type', function(){
+                var elem1 = params[0];
+                expect(typeof elem1).to.eql('string');
+                expect(params[0]).to.have.length(6);
+            });
+            it('should return string concatenated with unique id', function(){
+                expect(_.uniqueId.apply(_, params)).to.eql('value_1');
+                expect(_.uniqueId.apply(_, params)).to.eql('value_2');
+                expect(_.uniqueId.apply(_, params)).to.eql('value_3');
+            });
+        });
+
+        describe('uniqueId2', function() {
+            var params;
+            before(function(){
+                params = datasets.uniqueId2();
+            });
+            it('should check type', function(){
+                var elem1 = params[0];
+                expect(typeof elem1).to.eql('string');
+                expect(params[0]).to.have.length(10);
```

Comment:
Same as above. Does this additional test for same lodash method show user anything more?